LAKMobile/src/components/ModalAlert.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import { Modal, Pressable, StyleSheet, Text, View } from "react-native";
+
+interface ModalProps {
+  title: string;
+  message?: string;
+  confirmLabel: string;
+  onConfirm: () => void;
+  cancelLabel?: string;
+  onCancel?: () => void;
+  visible: boolean;
+}
+
+/**
+ * This component works like the built-in `Modal` (https://reactnative.dev/docs/modal).
+ *
+ * The `message`, `cancelLabel`, and `onCancel` props are optional. If
+ * `cancelLabel` is not provided, then there is only the confirmation button.
```

Comment:
Great job including this header comment, will enforce this for other major components going forward.